src/index.ts

```diff
@@ -170,6 +208,27 @@ export interface RegisterUserWithBiometricsParamType {
   reliantGUID: string;
   programGUID: string;
   consentID: string;
+  modalities: {
```

Comment:
Can this be an enum list to mimic the Android Native version?

Reply:
This is not possible, see [here](https://reactnative.dev/docs/native-modules-android#argument-types)